src/btree/row_srch.c

```diff
@@ -781,23 +781,28 @@ __wt_row_random_leaf(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt)
 }
 
 /*
- * __wt_row_random_descent --
- *	Find a random leaf page in a row-store tree.
+ * __wt_random_descent --
+ *	Find a random leaf page in a tree.
  */
 int
-__wt_row_random_descent(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt)
+__wt_random_descent(WT_SESSION_IMPL *session, WT_REF **refp, bool eviction)
```

Comment:
My vote would be for this to be in a different location. Either a new file; maybe ``src/btree/btree_random.c`` or maybe into ``bt_cursor.c``